src/routes/dashboard/helpers/channels.ts

```diff
@@ -0,0 +1,75 @@
+import type { Question, Questions } from '../types'
+
+export function sortChannels(questions: Question[]) {
+  const counts = questions.reduce((count, question) => {
+    const key = question.channelName
+    return count[key] ? ++count[key] : (count[key] = 1), count
+  }, {})
+
+  return Object.entries(counts)
+    .map(([channelName, count]) => {
+      return { group: channelName, count }
+    })
```

Comment:
```suggestion
    .map(([channelName, count]) => ({
      group: channelName, 
      count
    })
```

minor suggestion: [implicit return](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#function_body)